src/xml/ConfigParser.cpp

```diff
@@ -169,7 +171,7 @@ int ConfigParser::readXmlFile(std::string const &filePath)
   SAXHandler.error          = OnErrorFunc;
   SAXHandler.fatalError     = OnFatalErrorFunc;
 
-  std::ifstream ifs(filePath);
+  std::ifstream ifs{std::string(filePath).c_str()};
```

Comment:
I don't think this is worth it. Let's change the `filePath` back to `std::string`.

The `ifstream` accepts `std::string`, as it guarantees to be zero-terminated, which the system C-APIs require. `string_view` doesn't fullfill this guarantee.